src/keystore/sdcard.py

```diff
@@ -76,9 +70,7 @@ async def save_mnemonic(self):
         fullpath = "%s/%s.%s" % (path, self.fileprefix(path), filename)
 
         if fullpath.startswith(self.sdpath):
-            if not platform.is_sd_present():
-                raise KeyStoreError("SD card is not present")
-            platform.mount_sdcard()
+            platform.sdcard.mount()
 
         if platform.file_exists(fullpath):
```

Comment:
@stepansnigirev "File already exists" and "Would you like to overwrite this file?" feels a bit too weak here as a warning for potentially overwriting a seed by accident. Looks to me that we can safely assume that the user is overwriting a seed here with this check:` if platform.file_exists(fullpath)`

What do you think?